recipes/rgi/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "rgi" %}
+{% set version = "4.0.0" %}
+{% set sha256 = "17abb2e89608b3d82a491d3b8e894456da6d3c955df6e1563ae9240b02b3d0d1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+
+  url: https://card.mcmaster.ca/download/1/software-v{{ version }}.tar.gz
+  fn: card-software.tar.bz2
```

Comment:
I think `rgi_{{ version }}.tar.gz` is a better option here to avoid collisions between updated versions.

Reply:
I'm not sure I understand how their download folder is organized. The full address for the rgi software is https://card.mcmaster.ca/download/1/software-v4.0.1.tar.gz that downloads a file called card-software.tar.bz2. Then when you extract this file (on the build file) you get the more standard rgi-4.0.0.tar.gz. I'm not sure I can get access directly to this tar file but I will try.